src/restLink.ts

```diff
@@ -1,35 +1,29 @@
 import {
-  OperationTypeNode,
-  OperationDefinitionNode,
-  FragmentDefinitionNode,
-  // Query Nodes
   DirectiveNode,
   DocumentNode,
   FieldNode,
+  FragmentDefinitionNode,
+  OperationDefinitionNode,
+  OperationTypeNode,
   SelectionSetNode,
 } from 'graphql';
+import { Observable } from '@apollo/client/core';
+import { ApolloLink } from '@apollo/client';
```

Comment:
```suggestion
import { ApolloLink, Observable } from '@apollo/client';
```

As of v4, `@apollo/client` and `@apollo/client/core` are [aliases for each other](https://github.com/apollographql/apollo-client/blob/d0b8a7ef10a6551e90e8bf5e6b705a30453b39ad/package.json#L33-L35). We are recommending that users and other dependencies switch to `@apollo/client` as we plan to remove `@apollo/client/core` in v5. v4 moved all the React-related exports to `@apollo/client/react` so you won't have the "React doesn't exist" problem that you did in v3 when importing from `@apollo/client`.